code/__DEFINES/time.dm

```diff
@@ -0,0 +1,51 @@
+#define MIDNIGHT_ROLLOVER		864000	//number of deciseconds in a day
+
+#define JANUARY		1
+#define FEBRUARY	2
+#define MARCH		3
+#define APRIL		4
+#define MAY			5
+#define JUNE		6
+#define JULY		7
+#define AUGUST		8
+#define SEPTEMBER	9
+#define OCTOBER		10
+#define NOVEMBER	11
+#define DECEMBER	12
+
+//Select holiday names -- If you test for a holiday in the code, make the holiday's name a define and test for that instead
+#define NEW_YEAR				"New Year"
+#define VALENTINES				"Valentine's Day"
+#define APRIL_FOOLS				"April Fool's Day"
+#define EASTER					"Easter"
+#define HALLOWEEN				"Halloween"
+#define CHRISTMAS				"Christmas"
+#define FESTIVE_SEASON			"Festive Season"
+
+/*
+
+Days of the week to make it easier to reference them.
+
+When using time2text(), please use "DDD" to find the weekday. Refrain from using "Day"
+
+*/
+
+#define MONDAY		"Mon"
+#define TUESDAY	"Tue"
+#define WEDNESDAY	"Wed"
+#define THURSDAY	"Thu"
+#define FRIDAY		"Fri"
+#define SATURDAY	"Sat"
+#define SUNDAY		"Sun"
+
+#define SECONDS *10
```

Comment:
This is a good change, since it mirrors /tg/'s organization, but requires `#include "code\__HELPERS\time.dm"` to be cleaned of duplicate definitions, such as `SECONDS` and `MINUTES`.
Ideally in its own PR, though.